src/components/SearchableMultiSelect/SearchableMultiSelect.tsx

```diff
@@ -409,11 +542,16 @@ const SearchableMultiSelect = createClass({
 				</span>
 			),
 		];
-	},
-
-	renderOption({ optionProps, optionIndex }) {
+	};
+
+	renderOption = ({
+		optionProps,
+		optionIndex,
+	}: {
+		optionProps: ISearchableMultiSelectOptionProps;
+		optionIndex: number;
+	}): any => {
```

Comment:
This could have a stricter type than `any`, I think `React.ReactElement` would work.